mrjob/tools/emr/terminate_idle_job_flows.py

```diff
@@ -93,11 +109,23 @@ def main(cl_args=None):
         pooled_only=options.pooled_only,
         max_mins_locked=options.max_mins_locked,
         quiet=(options.quiet > 1),
+        **runner_kwargs(options)
     )
 
 
-def inspect_and_maybe_terminate_job_flows(
-    conf_paths=None,
+def runner_kwargs(options):
+    kwargs = options.__dict__.copy()
+    for unused_arg in ('quiet', 'verbose', 'max_hours_idle',
+                       'max_mins_locked',
+                       'mins_to_end_of_hour', 'unpooled_only',
+                       'pooled_only', 'pool_name', 'dry_run',
+                       'test'):
+        del kwargs[unused_arg]
```

Comment:
The advantage of the blacklist is that it matches up with options elsewhere in the file, so it's more likely to be updated.

Honestly, this is better than it was before, but not as maintainable as I'd like. This was the best I could do in the context of an emergency fix.

Filed a ticket on this anyway: #1097